### ISSUE TITLE
Is that possible to make the slider accessible by the keyboard tab?

### ISSUE THREAD

**Issue by cevicher0** [NightCatSama/vue-slider-component#470]
#### Describe the feature

Hi, I have used `vue-slider-component` in my project. We are currently focusing the web accessibility improvement. Now, the slider is not accessible by a keyboard tab.

#### Describe the solution you'd like

What I would like to achieve is that the slider should be accessible and adjustable by using the keyboard tab. I think we should simply add slider role and aria-valuenow etc. This page is the one I am expected. [http://www.oaa-accessibility.org/examplep/slider1/](http://www.oaa-accessibility.org/examplep/slider1/)

Thanks in advance.

**Comment by NightCatSama** [comment]
duplicate #402 , thanks for your suggestion.

**Comment by NightCatSama** [comment]
This feature is already supported in version 3.2.1. 😄 

[CHANGELOG](https://github.com/NightCatSama/vue-slider-component/blob/master/CHANGELOG.md#v321)
